shell/common/shell.cc

```diff
@@ -1490,6 +1499,7 @@ void Shell::LoadDartDeferredLibrary(
     intptr_t loading_unit_id,
     std::unique_ptr<const fml::Mapping> snapshot_data,
     std::unique_ptr<const fml::Mapping> snapshot_instructions) {
+  // TODO(jonahwilliams): what thread is this called from?
```

Comment:
I'd check the headerdoc in platform_view.h to be sure. But I think this can be called on any thread by the embedder. I also don't think this is used at all on non-Fuchsia platforms. And even there, I think we moved away from deferred libraries split into units. But, the code to service such a path may still be there.

Reply:
It doesn't say. This isn't for fuchsia, its for the android app bundle/deferred library support.

Reply:
it looks like this is probably only called from the platform thread, but I'm not sure if we have any real tests here so I'll leave it as is.